index.html

```diff
@@ -0,0 +1,1305 @@
+            </div>
+          </div>
+          <div class="card story-card mt-4 sticky-top">
+            <img
+              src="https://d.newsweek.com/en/full/1571844/bernie-sanders-joe-biden-missouri-primary-poll.jpg?w=397&amp;h=265&amp;l=49&amp;t=34&amp;q=90&amp;f=7b3fdc0e203af4625a2cc7333316e9c1"
+              alt="Bernie Sanders Joe Biden Missouri primary poll"
+              title="Bernie Sanders Joe Biden Missouri primary poll" class="mapping-sm_md card-img-top img-fluid">
+            <div class=" card-body p-0">
+              <h5 class="card-title font-weight-bold pt-2">
+                Biden's Missouri Lead Over Sanders Fueled by Over-50s and Non-Whites: Poll
+              </h5>
+              <p class="card-text mb-0">The former vice president is ahead of the Vermont senator in Missouri, a poll
+                shows, but his lead is within the margin of error.</p>
+            </div>
+            <div class="card-footer bg-white pl-0 story-footer font-weight-bold ">
+              Trump Says Sanders Would Be A 'Tougher' Opponent Than Frontrunner Joe Biden
+            </div>
+          </div>
+        
+      </div>
+
+
+      <div class="col-12 col-lg-6 col-md-6 ">
+        <div class="col-12">
+          <div class="row">
+            <h5 class="text-muted mt-4">TOP STORY</h5>
+            <div class="card pl-md-0">
+              <img
+                src="https://d.newsweek.com/en/full/1571844/bernie-sanders-joe-biden-missouri-primary-poll.jpg?w=397&amp;h=265&amp;l=49&amp;t=34&amp;q=90&amp;f=7b3fdc0e203af4625a2cc7333316e9c1"
+                alt="Bernie Sanders Joe Biden Missouri primary poll"
+                title="Bernie Sanders Joe Biden Missouri primary poll"
+                class="mapping-sm_md top-img card-img-top img-fluid">
+              <div class="card-body p-0">
+                <h5 class="card-title font-weight-bold pt-2">
+                  Biden's Missouri Lead Over Sanders Fueled by Over-50s and Non-Whites: Poll
+                </h5>
+                <p class="card-text mb-3">The former vice president is ahead of the Vermont senator in Missouri, a poll
+                  shows, but his lead is within the margin of error.</p>
+              </div>
+            </div>
+
+            <h5 class="text-muted mt-3 col-sm-12">CULTURE & TRAVEL</h5>
+            <div class="card-deck">
+              <div class="card travel w-100 ">
+                <img src="images/deck_img.jpg" alt="CUL_Uncharted_Stargazing_Banner"
+                  class="mapping-mobile lazy lazyloaded img-fluid w-100">
+                <div class="card-body pl-0">
+                  <h4 class="card-title culture-text pl-0">Best Stargazing Spots Around the Globe</h4>
+                  <p class="card-text pl-0">Astrotourism is taking off, and these are the best places—on our planet, at
+                    least—to witness the dazzling display of luminous balls of gas that we call stars</p>
+                </div>
+              </div>
```

Comment:
Please fix this section in the medium breakpoint.  The text is too big and the images smaller.
Yours:
![Screenshot from 2020-03-14 18-31-47](https://user-images.githubusercontent.com/51926937/76692877-59d0d180-6622-11ea-8431-53b4aa5da76b.png)

Original:
![Screenshot from 2020-03-14 18-31-54](https://user-images.githubusercontent.com/51926937/76692878-5dfcef00-6622-11ea-9689-d4dd12329813.png)

In the small breakpoint, this section should display in 100% width. Yours looks like this:
![Screenshot from 2020-03-14 18-45-27](https://user-images.githubusercontent.com/51926937/76692990-21ca8e00-6624-11ea-8c77-c68c640edba5.png)

The original:
![Screenshot from 2020-03-14 18-45-38](https://user-images.githubusercontent.com/51926937/76692996-25f6ab80-6624-11ea-91d3-ea42af88fc2e.png)

Reply:
fixed the card deck section by making text smaller